src/components/DeviceAction/rendering.tsx

```diff
@@ -0,0 +1,550 @@
+})``;
+
+const TitleContainer = styled(Flex).attrs({
+  py: 8,
+})``;
+
+const TitleText = ({ children }: { children: React.ReactNode }) => (
+  <TitleContainer>
+    <Log>{children}</Log>
+  </TitleContainer>
+);
+
+const DescriptionText = styled(CenteredText).attrs({
+  variant: "bodyLineHeight",
+  py: "8px",
+  fontWeight: "medium",
+  color: "neutral.c70",
+})``;
+
+const ConnectDeviceNameText = styled(Tag).attrs({
+  my: "8",
+})``;
+
+const StyledButton = styled(Button).attrs({
+  mt: 6,
+  alignSelf: "stretch",
+})``;
+
+const ConnectDeviceExtraContentWrapper = styled(Flex).attrs({
+  mb: 8,
+})``;
+
+type RawProps = {
+  t: (key: string, options?: { [key: string]: string | number }) => string;
+  colors?: any;
+  theme?: "light" | "dark";
+};
+
+export function renderRequestQuitApp({
+  t,
+  device,
+  theme,
+}: RawProps & {
+  device: Device;
+}) {
+  return (
+    <Wrapper>
+      <AnimationContainer>
+        <Animation
+          source={getDeviceAnimation({ device, key: "quitApp", theme })}
+        />
+      </AnimationContainer>
+      <CenteredText>{t("DeviceAction.quitApp")}</CenteredText>
+    </Wrapper>
+  );
+}
+
+export function renderRequiresAppInstallation({
+  t,
+  navigation,
+  appNames,
+}: RawProps & {
+  navigation: any;
+  appNames: string[];
+}) {
+  const appNamesCSV = appNames.join(", ");
+
+  return (
+    <Wrapper>
+      <CenteredText>
+        {t("DeviceAction.appNotInstalled", {
+          appName: appNamesCSV,
+          count: appNames.length,
+        })}
+      </CenteredText>
+      <ActionContainer>
+        <StyledButton
+          event="DeviceActionRequiresAppInstallationOpenManager"
+          type="primary"
+          title={t("DeviceAction.button.openManager")}
+          onPress={() =>
+            navigation.navigate(NavigatorName.Manager, {
+              screen: ScreenName.Manager,
+              params: { searchQuery: appNamesCSV },
+            })
+          }
+        />
+      </ActionContainer>
+    </Wrapper>
+  );
+}
+
+export function renderVerifyAddress({
+  t,
+  device,
+  currencyName,
+  onPress,
+  address,
+  theme,
+}: RawProps & {
+  device: Device;
+  currencyName: string;
+  onPress?: () => void;
+  address?: string;
+}) {
+  return (
+    <Wrapper>
+      <AnimationContainer withVerifyAddressHeight={device.modelId !== "blue"}>
+        <Animation
+          source={getDeviceAnimation({ device, key: "validate", theme })}
+        />
+      </AnimationContainer>
+      <TitleText>{t("DeviceAction.verifyAddress.title")}</TitleText>
+      <DescriptionText>
+        {t("DeviceAction.verifyAddress.description", { currencyName })}
+      </DescriptionText>
+      <ActionContainer>
+        {onPress && (
+          <StyledButton
+            event="DeviceActionVerifyAddress"
+            type="primary"
+            title={t("common.continue")}
+            onPress={onPress}
+          />
+        )}
+        {address && <TitleText fontWeight="bold">{address}</TitleText>}
```

Comment:
The fontWeight is misleading, it doesn't seem to do anything

Reply:
good eyes haha thanks